content/updates/2020-05/1-new-connection-monitoring-page.md

```diff
@@ -0,0 +1,30 @@
+# New: monitor products errors on your connections
+
+## A new page to monitor your connection errors
+::: meta-data type="New" features="Connections" available="mid-May" link-to-doc="../articles/manage-your-connections.html#monitor-a-connection"
+:::
+
+This feature should help you to reduce the time needed to solve issues. The goal of displaying product business errors is to allow you to solve them before calling for IT support or your integrator support.
+
+In other words, why a product hasn't been created or updated by your source connection?
+
+![Connection monitoring page](../img/new-connection-monitoring-page.png)
+
+## New metrics on the connection dashboard
+::: meta-data type="New" features="Connections" available="late-May" link-to-doc="../articles/manage-your-connections.html#monitor-a-connection"
+:::
+
+The connection dashboard now displays: 
+- a new graph with the number of product errors (business and technical) for the past week,
+- and connection cards with the number of business errors during the past week with a link to the errors list of the related connection.
```

Comment:
What are "connections cards"?

Reply:
We added "cards" on the dashboard. One for each source connection that displays connection icon, name, # of business errors for the last 7 days. I added the screenshot to illustrate.